__device-tests__/gutenberg-editor-sanity-test-1-visual.test.js

```diff
@@ -331,5 +331,54 @@ describe( 'Gutenberg Editor - Test Suite 1', () => {
 			);
 			expect( screenshot ).toMatchImageSnapshot();
 		} );
+
+		it( 'should display the block outline of a block inserted from the appender', async () => {
```

Comment:
While I recognize the block outline is a highly visual element, I might argue that while its presence is important its _style_ is not critical. I posit we could/should merely assert the functionality succeeds in the form of asserting the outline's presence in a traditional integration test. I.e. we could add a `testID` to the [associated element](https://github.com/WordPress/gutenberg/blob/7c7563ac875cf941fe1b1ae1715d3604c5af5bac/packages/block-editor/src/components/block-list/block-outline.native.js#L57) and assert its presence rather than incurring the cost of an e2e visual snapshot test. 

WDYT?

Reply:
I did consider the expense of the visual test, but I had not considered that the same behavior could be tested with an integration test asserting the presence of BlockOutline. As this current visual test addition asserts the behavior fixed in https://github.com/wordpress-mobile/gutenberg-mobile/pull/6133, I suggest we keep the visual test and I can replace it with an integration test shortly afterward. Wydt?

Reply:
Sounds good. 👍🏻